Content.Client/Shuttles/UI/DockObject.xaml.cs

```diff
@@ -1,23 +1,22 @@
-using System.Text;
 using Content.Shared.Shuttles.BUIStates;
-using Content.Shared.Shuttles.Systems;
 using JetBrains.Annotations;
 using Robust.Client.AutoGenerated;
 using Robust.Client.Graphics;
 using Robust.Client.UserInterface.Controls;
 using Robust.Client.UserInterface.XAML;
-using Robust.Shared.Map;
 
 namespace Content.Client.Shuttles.UI;
 
 [GenerateTypedNameReferences]
 public sealed partial class DockObject : PanelContainer
 {
+#pragma warning disable CS0067
     [PublicAPI]
     public event Action? UndockPressed;
 
     [PublicAPI]
     public event Action? ViewPressed;
+#pragma warning restore CS0067
```

Comment:
Should these be kept around? There's nothing actually calling these so surely the warning is valid and it shouldn't just be suppressed?

Reply:
I went off the fact they already had `[PublicAPI]` with an apparent meaning of "please do not remove this", but I'm happy to trash them anyway on your say-so.